src/shared/hooks/use-bottom-sheet-drag.ts

```
@@ -0,0 +1,82 @@
+import { useEffect, useRef } from 'react';
+
+interface UseBottomSheetDragProps {
+  sheetRef: React.RefObject<HTMLDivElement | null>;
```

ESLint

[error] 'React' is not defined.

(no-undef)